src/barcodes/CODE128/helpers.js

```diff
@@ -0,0 +1,21 @@
+const OPTIONS = [ 'ean128', 'ignoreBrackets', 'ignoreSpaces' ];
+
+export const normalizeOptions = (options) => {
+	OPTIONS.forEach((option) => {
+		if (option in options) {
+			const value = options[option];
+			options[option] = value === true || value === 'true';
+		}
+	});
+	return options;
+};
+
+export const formatData = (data, options) => {
+	if (options.ignoreBrackets === true) {
```

Comment:
Is it really neccacary to include these options? Sure, it would be a small convinience for people that want to use it in this exact way but it's not that hard to do outside of the library 🙂 

Doing this:
````javascript
JsBarcode(barcode, data.replace(/[()]/g), {
  text: data
})
````

Instead of this:
````javascript
JsBarcode(barcode, data), {
  ignoreBrackets : true
})
````